packages/waku/src/router/client.ts

```diff
@@ -48,15 +48,19 @@ const normalizeRoutePath = (path: string) => {
   return path;
 };
 
-const parseRoute = (url: URL): RouteProps => {
+const parseRoute = (url: URL, ignore_hash: boolean = false): RouteProps => {
```

Comment:
Just for preference:
```suggestion
const parseRoute = (url: URL, ignoreHash = false): RouteProps => {
```